docker/clone-and-preflight.ts

```diff
@@ -64,7 +64,7 @@ if (projectUsesPostgresql) {
   //
   // Example script:
   // https://github.com/upleveled/preflight-test-project-next-js-passing/blob/e65717f6951b5336bb0bd83c15bbc99caa67ebe9/scripts/alpine-postgresql-setup-and-start.sh
-  const postgresUid = Number(await execa`id -u postgres`);
+  const postgresUid = Number((await $`id -u postgres`).stdout);
```

Comment:
looks like the `execa` identifier isn't just the one imported directly from the `'execa'` package, but it's this:

```ts
const projectPath = 'project-to-check';
const execa = bindExeca({ cwd: projectPath });
```

so probably we need to stick with using the `execa` identifier, or do a bigger change to switch everything to `$`, with this binding

Reply:
Changed to `execa`. [cfc6c4e](https://github.com/upleveled/preflight/pull/629/commits/cfc6c4e6009bb9370c58325ffb656c7839fe911f)